.github/workflows/publish-next.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         name: Setup Build Environment
       - uses: actions/setup-node@v1
         with:
-          node-version: 12
+          node-version: 14
```

Comment:
I believe that this workflow is obsolete and should be removed. See: https://github.com/eclipse-theia/vscode-builtin-extensions/issues/51

Instead of updating it, do you want to cover this issue in this PR? I think you can also remove related script `src/publish.js`.

Reply:
Oh, and I guess if we go ahead, we can also remove the README section about that workflow

Reply:
Good point, although I would prefer to keep a single purpose for each change, so I don't mind following up with an additional PR on top of this one.
This way the commit message will be clearer and we could address comments and revert changes (if needed) more specifically.
Let me know what you think!

Reply:
One way to achieve a better separation is to have a separate commit in the same PR. But a separate PR is also acceptable.

Reply:
The following PR was created to address issue: #51
https://github.com/eclipse-theia/vscode-builtin-extensions/pull/85

Reply:
After discussing this with Vince, we agreed to use a single PR, two commits, and the first one addressing the removal of the publish-next workflow.